tools/parsinginjs/src/main/java/java/nio/ByteBuffer.java

```diff
@@ -0,0 +1,23 @@
+package java.nio;
+
+import org.enso.parsinginjs.ByteBufferImpl;
+
+public abstract class ByteBuffer {
```

Comment:
Emulation of `ByteBuffer` as [Bck2Brwsr VM](https://github.com/jtulach/bck2brwsr/) doesn't provide one yet.